### PR TITLE
Peer to Peer code project

### PR DESCRIPTION
**Pull request by alexiscyber14**
Project requirements:
 -The name field accepts up to 30 chars.
 -The email field accepts only emails.
-The message field accepts up to 500 chars.
-All fields have placeholder text.
-All fields have the correct type.
-A user cannot submit the form if one or more of the fields is empty.
-Collect data by using the [Formspree service](https://formspree.io/html).
-You don't need to implement any functionality that requires JavaScript.
-Each group member needs to open a new pull request in their repo.

Achievements:
-Worked on the project with my coding partner @AmineDerbal 
-Added the contact form
-Used from spree data collecting function
-used only CSS and HTML
-No linter errors
 

Any feedback will be appreciated. thank you!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,7 +178,27 @@ <h4>Skills</h4>
...
-
+<section class="container contact">
+    <div class="contact-text">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p> 
+    </div>
+    <div class="edge">
+        <form action="https://formspree.io/f/mayznewd" class="form" action="#" method="POST">
+           <input type="text" placeholder="Full name" required  maxlength="30">
+           <input type="email" placeholder="Email address" required>
+           <textarea name="text-area" id="" cols="30" rows="10" maxlength="500">Enter text here</textarea>
```
Comment (binhussen) [inline index.html]: - Kindly, your All input fields have to have the required attribute to activate the validation on submission 👌 so please add the required for this `textarea` also👌👍.

File: index.html [hunk 2]
```diff
@@ -178,7 +178,27 @@ <h4>Skills</h4>
...
+           <button id="submit">Get in touch</button>
```
Comment (binhussen) [inline index.html:190]: - [Optional] Kindly, your `button` hasn't `type` attribute 👌👍 I think it is better to use the `button` element with the type `submit`👍👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @alexiscyber14 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @alexiscyber14 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - All linter checks passed ✔️
 - Descriptive pull request ✔️
 - Used GitHub flow ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**